src/vm/eventpipemetadatagenerator.h

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __EVENTPIPE_METADATAGENERATOR_H__
+#define __EVENTPIPE_METADATAGENERATOR_H__
+
+#ifdef FEATURE_PERFTRACING
+
+enum class EventPipeEventLevel;
+
+enum class EventPipeParameterType
```

Comment:
could you add some comment about where these values come from? (sth like "this enum is TypeCode + Guid")

Reply:
Yes, will do.  Thanks for pointing this out.